tools/topology/topology2/sof-hda-generic.conf

```diff
@@ -58,6 +58,10 @@ Define {
 	HDMI1_ID	1
 	HDMI2_ID	2
 	HDMI3_ID	3
+	HDMI1_PCM_ID	3
+	HDMI2_PCM_ID	4
+	HDMI3_PCM_ID	5
+	HDMI4_PCM_ID	6
```

Comment:
IIRC we never enabled 4 streams for HDMI in sof-hda-generic, did we?

And also I am confused by the HDMI1_ID being 1, is this correct? I thought we had the HDaudio codec Dailink first, and HDMI last?

Reply:
> IIRC we never enabled 4 streams for HDMI in sof-hda-generic, did we?

I will remove HDMI4_PCM_ID from sof-hda-generic.conf.

> 
> And also I am confused by the HDMI1_ID being 1, is this correct? I thought we had the HDaudio codec Dailink first, and HDMI last?

No, we have HDMI first. see [skl_hda_be_dai_links](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/intel/boards/skl_hda_dsp_common.c#L82)
```
iDisp1
iDisp2
iDisp3
Analog Playback and Capture
Digital Playback and Capture
dmic01
dmic16k
```

Reply:
ok